src/material-experimental/mdc-form-field/directives/prefix.ts

```diff
@@ -17,7 +17,8 @@ export const MAT_PREFIX = new InjectionToken<MatPrefix>('MatPrefix');
 
 /** Prefix to be placed in front of the form field. */
 @Directive({
-  selector: '[matPrefix]',
+  /** @breaking-change 13.0.0 remove [matPrefix] */
```

Comment:
Not sure whether leaving a `@breaking-change` here is correct. This means that when v13 comes around, we can delete it, but we have to keep it as long as the non-MDC version is around.

Reply:
Yeah I guess you're right, removed it. I wanted to put `@deprecated` on it, but that would make the whole directive deprecated.